src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-style: book;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-weight: 300;
+}
+
+form[data-qa="big"] {
+  box-sizing: border-box;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  margin: 20px 0;
+  padding: 0;
+  color: #3d4e61;
+}
+
+form[data-qa="big"]:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  color: #3d4e61;
+}
+
+form[data-qa="big"]:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: Avenir, sans-serif;
+  font-style: heavy;
+  border-radius: 3px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+input[data-qa="keypress"] {
+  width: 100%;
+  padding: 0;
+  border: 0;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  font-style: book;
+  font-weight: 300;
+  outline: 0;
+}
+
+.icon-big {
+  height: 19px;
+  padding: 25px 17px 26px 26px;
+}
+
+form[data-qa="small"] {
+  box-sizing: border-box;
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  color: #3d4e61;
+  padding: 0;
+}
+
+form[data-qa="small"]:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+form[data-qa="small"]:focus {
```

Comment:
This rule is targeting the whole form instead of the input itself. Use .smallSearch:focus to target the input directly.